flows/constants.py

```diff
@@ -1,21 +1,48 @@
-CARBON_SUBGRAPH_URL = (
-    "https://api.thegraph.com/subgraphs/name/klimadao/polygon-digital-carbon"
-)
-CARBON_LEGACY_SUBGRAPH_URL = (
-    "https://api.thegraph.com/subgraphs/name/klimadao/polygon-bridged-carbon"
-)
-CARBON_ETH_SUBGRAPH_URL = (
-    "https://api.thegraph.com/subgraphs/name/klimadao/ethereum-bridged-carbon"
-)
-CARBON_CELO_SUBGRAPH_URL = (
-    "https://api.thegraph.com/subgraphs/name/klimadao/celo-bridged-carbon"
-)
-CARBON_HOLDERS_SUBGRAPH_URL = (
-    "https://api.thegraph.com/subgraphs/name/klimadao/klimadao-user-carbon"
-)
-PAIRS_SUBGRAPH_URL = (
-    "https://api.thegraph.com/subgraphs/name/klimadao/klimadao-pairs"
-)
+import os
+
+GRAPH_API_KEY = os.environ.get('GRAPH_API_KEY')
+GRAPH_BASE_URL = f'https://gateway-arbitrum.network.thegraph.com/api/{GRAPH_API_KEY}/subgraphs/id/'
```

Comment:
Should you not be using this in the `production` configuration?

Reply:
yes we will, this is still a draft